include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_radix_sort_one_wg.h

```diff
@@ -152,7 +152,7 @@ struct __subgroup_radix_sort
     struct __one_group_submitter;
 
     template <typename... _Name>
-    struct __one_group_submitter<__internal::__optional_kernel_name<_Name...>>
+    struct __one_group_submitter<oneapi::dpl::__par_backend_hetero::__internal::__optional_kernel_name<_Name...>>
```

Comment:
This is only one place inside all this file where according to comments from this file
```C++
//The file is an internal file and the code of that file is included by a major file into the following namespaces:
//namespace oneapi
//{
//namespace dpl
//{
//namespace __par_backend_hetero
//{
```
we simple specify `__internal` for namespace `oneapi::dpl::__par_backend_hetero::__internal`

In some another lines from this file we using full namespace.
Example:
```C++
        using _SortKernelLoc = oneapi::dpl::__par_backend_hetero::__internal::__kernel_name_provider<
            __radix_sort_one_wg_kernel<_KernelNameBase, __wg_size_t, __block_size_t, __call_0_t>>;
```